OutOfSchool/OutOfSchool.AuthCommon/Extensions/AuthCommonServiceExtensions.cs

```diff
@@ -31,7 +32,8 @@ public static void AddAuthCommon(this IServiceCollection services, Configuration
         services.AddEmailSender(
             isDevelopment,
             mailConfig.SendGridKey,
-            builder => builder.Bind(config.GetSection(EmailOptions.SectionName)));
+            builder => builder.Bind(config.GetSection(EmailOptions.SectionName)))
+            .AddEmailRendererConfiguration(new EmailContentConfig(config.GetSection("Identity")["Authority"]));
```

Comment:
(Code style) Please consider storing string literals in related classes (see f.e. `EmailOptions.SectionName` one line above). Especially, when they are reused more than twice